deepchem/feat/__init__.py

```diff
@@ -25,6 +25,7 @@
 from deepchem.feat.molecule_featurizers import MordredDescriptors
 from deepchem.feat.molecule_featurizers import Mol2VecFingerprint
 from deepchem.feat.molecule_featurizers import MolGraphConvFeaturizer
+from deepchem.feat.molecule_featurizers import molgan_featurizer
```

Comment:
I'd suggest renaming `MolganFeaturizer` to match the naming convention for the other featurizers

Reply:
Ah looking below I think this is a typo since the class is `MolGanFeaturizer`